org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/tree/AbstractTree.java

```diff
@@ -2768,7 +2747,8 @@ public void setNodesCheckedFromUI(List<ITreeNode> nodes, boolean checked) {
           setTreeChanging(true);
           nodes = resolveNodes(nodes);
           if (!nodes.isEmpty()) {
-            setNodesChecked(nodes, checked, true);
+            // With isAutoCheckChildNodes() it's possible to check disabled nodes from UI
```

Comment:
Until now, it was not possible to check a disabled node from UI. Since the autoCheckChildren-change, disabled nodes can be checked by checking all child nodes of a disabled node. I have added more details to the comment.